inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/fully_connected_gpu_bfyx_ref.cl

```diff
@@ -27,6 +27,27 @@ KERNEL(fc)(
 #endif
     )
 {
+#if OUTPUT_3D
+    const uint oxfm = get_global_id(0);
+    const uint b = get_global_id(1);
+    const uint oym = oxfm % OUTPUT_SIZE_Y;
+    const uint ofm = oxfm / OUTPUT_SIZE_Y;
+
+    ACCUMULATOR_TYPE dotProd = (ACCUMULATOR_TYPE)0;
```

Comment:
I suppose you could use `ACCUMULATOR_TYPE dotProd = ACCUMULATOR_VAL_ZERO;` here.

Reply:
Done